support-frontend/assets/helpers/subscriptionsForms/formActions.ts

```diff
@@ -119,7 +129,10 @@ export type Action =
 			type: 'SET_SALESFORCE_CASE_ID';
 			caseId: string;
 	  }
-	| AddressAction
+	| {
+			type: 'ON_DELIVERY_COUNTRY_CHANGED';
+			country: string;
+	  }
```

Comment:
Previously this reducer was responding to the `SET_COUNTRY` action from the address reducer. That action is now coming from an rtk slice. This means we can't extract a nice type for it here (the `type` field in the generated action is just `string` as it is created dynamically). Instead, I've made a new `ON_DELIVERY_COUNTRY_CHANGED` action that is triggered in a side effect listener.

Reply:
👍 I think this is something we can ultimately handle in the product reducer by using `extraReducers` to listen for the address slice action, which should smooth this all out later.